src/main/features/claim/helpers/feesTableViewHelper.ts

```diff
@@ -1,4 +1,9 @@
 import { isUndefined } from 'util'
+import { FeesClient } from 'fees/feesClient'
+import { FeeRange as MergeableRange } from 'claim/helpers/feesTableViewHelper'
+import { FeeRange as ViewFeeRange } from 'fees/models/feeRange'
+
+const supportedFeeLimitInGBP: number = 10000
```

Comment:
please tell me this is the only place this is in?

Reply:
yup. moved it there as it is now needed twice